src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+const firstPromise = () =>
+  new Promise((resolve, reject) => {
+    const handleLeftClick = () => {
+      clearTimeout(timeoutId);
+      resolve('First promise was resolved');
+      window.removeEventListener('click', handleLeftClick);
+    };
+
+    const timeoutId = setTimeout(() => {
+      reject(new Error('First promise was rejected'));
+      window.removeEventListener('click', handleLeftClick);
+    }, 3000);
+
+    window.addEventListener('click', handleLeftClick);
+  });
+
+firstPromise()
+  .then(createSuccessMessage)
+  .catch((err) => createErrorMessage(err.message));
+
+const secondPromise = () =>
+  new Promise((resolve) => {
+    const handleAnyClick = (e) => {
+      e.preventDefault();
+      resolve('Second promise was resolved');
+    };
+
+    document.addEventListener('click', handleAnyClick, { once: true });
+    document.addEventListener('contextmenu', handleAnyClick, { once: true });
+  });
+
+secondPromise().then(createSuccessMessage);
+
+const thirdPromise = () =>
+  new Promise((resolve) => {
+    let leftClick = false;
+    let rightClick = false;
+
+    const handleLeftClick = (e) => {
+      e.preventDefault();
```

Comment:
The use of `e.preventDefault()` in the left click handler might not be necessary unless you specifically want to prevent the default action of the click event. Consider removing it if it's not needed.